README.rst

```diff
@@ -2,6 +2,15 @@
 Scrapy & Autoextract API integration
 ====================================
 
+.. image:: https://img.shields.io/badge/python-3.5-blue.svg
```

Comment:
It should be 3.5 - 3.7, not just 3.5; see e.g. how this is done in price-parser: https://raw.githubusercontent.com/scrapinghub/price-parser/master/README.rst

Reply:
Yes, not that we are on Pypi, there's not need to hardcode the Python version so I'll replace it.

Reply:
It seems we also need to add Python 3.5 to setup.py, as it is supported (we run tests for it)